src/cabal

```diff
@@ -69,7 +69,7 @@ initialize () {
 
     UNINITIALIZED=$(cd "$PROJECT_ROOT" && git submodule | grep "^-" | cut -d ' ' -f 2)
     for submodule in $UNINITIALIZED; do
-        git submodule update --init $submodule
+        git submodule update --init $PROJECT_ROOT/$submodule
```

Comment:
On ci this won't work because of amusingly old git version.

Need to:
```
$(cd "$PROJECT_ROOT" && git submodule update --init $submodule) || exit $?
```